docs/guides/allocation-limits.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Set instance allocation limits
+description: Set, view, and change how much time an instance is allowed to use.
+platform: cloud
+---
+# Set instance allocation limits
+
+Premium accounts are alloted a certain amount of time.  Because each account can have many instances, setting allocation limits helps you control which instances are allowed to consume that time.
```

Comment:
```suggestion
An account may have allotted time on the premium plan. Because each account can have many premium plan instances, setting allocation limits helps you control which instances are allowed to consume that time.
```